storage/storage.go

```diff
@@ -1333,6 +1377,40 @@ func newObject(o *raw.Object) *ObjectAttrs {
 	}
 }
 
+func newObjectFromProto(r *storagepb.WriteObjectResponse) *ObjectAttrs {
+	o := r.GetResource()
+	if r == nil || o == nil {
+		return nil
+	}
+	return &ObjectAttrs{
+		Bucket:                  o.Bucket,
```

Comment:
Will this now return the path including the _ for projectID? If so we should remove that probably and return bucket name only.

Reply:
Good point, it will. Done!